src/plugins/unified_histogram/public/layout/helpers.ts

```diff
@@ -12,14 +12,12 @@ const TRANSFORMATIONAL_COMMANDS = ['stats', 'keep'];
 
 export const shouldDisplayHistogram = (query: AggregateQuery) => {
   let queryHasTransformationalCommands = false;
```

Comment:
nit: worth make it become a simpler `.some` check?

```suggestion
  return !TRANSFORMATIONAL_COMMANDS.some( command => query.esql.toLowerCase().includes(command) );
```

Reply:
Me gusta https://github.com/elastic/kibana/pull/182079/commits/473788601ab912733db8511c24490ffd618c3410